ClangSharpPInvokeGenerator/Program.cs

```diff
@@ -89,12 +89,17 @@ public static int Run(InvocationContext context)
             arr = arr.Concat(defines.Select(x => "-D" + x)).ToArray();
             arr = arr.Concat(additionalArgs).ToArray();
 
+            var translationFlags = CXTranslationUnit_Flags.CXTranslationUnit_None;
```

Comment:
Also updated this to start passing in some flags. Namely, not processing function bodies currently and including implicit attribute information (which ensures we aren't missing anything if the user specifies additional command line options).